packages/web/components/cards/rewards-card.tsx

```diff
@@ -1,55 +1,71 @@
+import classNames from "classnames";
 import React from "react";
 
-import { Icon } from "~/components/assets";
+import { DynamicLottieAnimation } from "~/components/animation";
 import { Button } from "~/components/buttons";
 import { Tooltip } from "~/components/tooltip";
 
 export const RewardsCard: React.FC<{
   title: string;
-  tooltipContent: string;
   disabledTooltipContent?: string;
   onClick: () => void;
-  image?: JSX.Element;
   disabled: boolean;
+  globalLottieFileKey: string;
+  position: "left" | "right";
 }> = ({
   title,
-  tooltipContent,
   disabledTooltipContent,
   onClick,
-  image = null,
   disabled,
+  globalLottieFileKey,
+  position,
 }) => {
-  const ConditionalWrapper: React.FC<{ children: React.ReactNode }> = ({
+  const ConditionalTooltip: React.FC<{ children: React.ReactNode }> = ({
```

Comment:
Nit: I think children come out of the box with the FC type, and are of a more specific type than ReactNode.